src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynService.kt

```diff
@@ -138,20 +148,20 @@ fun UtgiftsperiodeDto.split(): List<UtgiftsMåned> {
     val perioder = mutableListOf<UtgiftsMåned>()
     var måned = this.periode.fom
     while (måned <= this.periode.tom) {
-        perioder.add(UtgiftsMåned(måned, this.barn, this.utgifter.toBigDecimal()))
+        perioder.add(UtgiftsMåned(måned, this.barn, this.utgifter.toBigDecimal(), this.aktivitetstype, this.periodetype))
         måned = måned.plusMonths(1)
     }
     return perioder
 }
 
 /**
- * Merger sammenhengende perioder hvor beløp og @BeløpsperiodeBarnetilsynDto#beregningsgrunnlag (it.toKey()) er like.
+ * Merger sammenhengende perioder hvor beløp, aktivitetstype, periodetype og @BeløpsperiodeBarnetilsynDto#beregningsgrunnlag (it.toKey()) er like.
```

Comment:
Hvorfor er det ønskelig å splitte beregnede perioder hvis aktivitet/periodetype er ulik?

Reply:
Fordi denne metoden brukes for å beregne ytelser. Hvis man ikke skiller på aktivitetstype og periodetype vil vel perioder med forskjellige aktivitetstyper/periodetyper merges sammen og noen av periodene vil få satt feil periodetype og/eller aktivitetstype?

Reply:
Samme sak som https://github.com/navikt/familie-ef-sak/pull/2021#discussion_r1072603368